docs/docs/user/references/components.md

```diff
@@ -12,6 +12,9 @@ This section explains the different components of KPOps, their usage and configu
 <!-- To learn more about KPOps' components hierarchy, visit the
 [architecture](/docs/developer/architecture/component-inheritance.md) page. -->
 
+!!! note "Environment-specific pipeline definitions"
+    Similarly to [defaults](../defaults/#configuration), it is possible to possible to have an unlimited amount of additional environment-specific pipeline definitions. The naming convention is the same: add a suffix of the form `_{environment}` to the filename. Learn more about environments in the [Config](../config/#__codelineno-0-10) section.
```

Comment:
duplicate wording in this sentence

Reply:
I only noticed the repeating "possible to". I've removed it.